ocaml/xapi-storage/generator/lib/control.ml

```diff
@@ -116,6 +122,11 @@ type volume = {
   ; keys: (string * string) list
         (** A list of key=value pairs which have been stored in the Volume
       metadata. These should not be interpreted by the Volume plugin. *)
+  ; volume_type: volume_type option [@default Some Data]
```

Comment:
The option type doesn't seemed used, because the default is not None.
This should be valid:
```suggestion
  ; volume_type: volume_type [@default Data]
```
Likewise for cbt_enabled